spec/models/token_spec.rb

```diff
@@ -182,4 +182,30 @@
       end
     end
   end
+
+  describe '#status' do
+    context 'when expired' do
+      let(:token) { create(:token, exp: 1.day.ago) }
+
+      it 'returns expired' do
+        expect(token.status).to eq('expired')
+      end
+    end
```

Comment:
Pour rendre ça plus joli:

```suggestion
  describe '#status' do
    subject { token.status }
    
    context 'when expired' do
      let(:token) { create(:token, exp: 1.day.ago) }

      it { is_expected.to eq('expired') }
    end
```

idem plus bas